cmd/res/device.virtual.binary.yaml

```diff
@@ -10,28 +10,13 @@ deviceResources:
     name: "Binary"
     description: "Generate random binary value"
     properties:
-      value:
-        { type: "Binary", readWrite: "R", mediaType: "random" }
-      units:
-        { type: "String", readWrite: "R", defaultValue: "random binary value" }
+      valueType: "Binary"
+      readWrite: "R"
+      mediaType: "random"
 
 deviceCommands:
```

Comment:
Since it is the same as the Device Resource, we can totally eliminate deviceCommands in this profile